src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+  let day = 1;
+
+  for (; day < days; day++) {
+    cost += 40;
+  }
+
+  if (day >= 3 && day <= 6) {
+    return (cost - 20);
+  } else if (day >= 7) {
+    return (cost - 50);
```

Comment:
What is the magic numbers 20, 50, 40?
Add variables
![2023-02-10_15h32_37](https://user-images.githubusercontent.com/61904995/218105080-b7a2721f-298c-4adc-9b1b-30dbe7a5f926.png)